tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2024 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Provides a library to deal with the CoCo KBS
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+kubernetes_dir="$(dirname "$(readlink -f "$0")")"
+# shellcheck disable=1091
+source "${kubernetes_dir}/../../gha-run-k8s-common.sh"
+
+# Where the kbs sources will be cloned
+readonly COCO_KBS_DIR="/tmp/kbs"
```

Comment:
Can you use `mktemp -d` here with a global variable and set it in a `setup()` function or similar.

Reply:
hi @jodh-intel , I also like the idea of using a temp dir, my only concern on this case is the added complexity. The functions will be called on different GHA steps, I don't think exported variables on one will be carried out to the next. I can think in a couple of options though:
 * save $COCO_KBS_DIR as an step output
 * save $COCO_KBS_DIR in $GITHUB_ENV, which is used by github actions to export variables
 
However both options works only on github context; running locally we will need another logic.

Worth tackling this problem?